### DISCUSSION TITLE
Question about SafeMath .div function

### DISCUSSION THREAD

**Discussion by 0xwtfwtf** [Cyfrin/foundry-full-course-cu#859] [General]
I'm looking at a contract that uses SafeMath and has code x.div(y)

Is x.div(y) equal to x/y or y/x?

Like if x=2 and y=3, does x.div(y) = 3/2 or 2/3??

**Comment by shikhar229169** [comment]
https://github.com/ConsenSysMesh/openzeppelin-solidity/blob/master/contracts/math/SafeMath.sol

Here, x will be the first argument, and y will be the second argument. So, it will be 2/3
